.github/workflows/deploy.yml

```diff
@@ -0,0 +1,29 @@
+---
+name: "deploy"
+
+# yamllint disable-line rule:truthy
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Cloning repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Push to dokku
+        uses: dokku/github-action@master
+        with:
+          git_remote_url: "ssh://dokku@azure.dev.codeforafrica.org/twoopstracker-ui"
```

Comment:
I think something like `ssh://dokku@dokku-1.fe.codeforafrica.org/twoopstracker-ui`, or `ssh://dokku@prod-1.fe.codeforafrica.org/twoopstracker-ui`, etc., are better names. We don't want to specify whether azure or aws, etc., and I think this isn't a dev instance but where we'll be doing prod deployments.

----

`fe` = front-end. `ui` would also work